velox/type/StringToFloatParser.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <fast_float/fast_float.h>
```

Comment:
Is this header needed? I assume we only need headers for Status and std:string_view, no?

Reply:
Yes, `fast_float::from_chars` is used in `velox/type/StringToFloatParser.cpp`

Reply:
@wypb Since the include is used needed in .cpp, then we can move it there. No need to keep it in the header file, right?

Reply:
Good suggestion, I'll move it to cpp file.